### ISSUE TITLE
Request: TPP1 mapper

### ISSUE THREAD

**Issue by red031000** [pokemon-speedrunning/gambatte-core#21]
Hello,

I am currently considering using the custom TPP1 mapper for a GB game, however very few emulators support it, gambatte speedrun is one of the most used emulators, with the advantage of being integrated into bizhawk as well, as such, if possible, I'd like to request that the TPP1 mapper be implemented

the specification for the mapper can be found here: https://github.com/aaaaaa123456789/tpp1/blob/master/specification.md

thank you for your time

**Comment by CasualPokePlayer** [comment]
I would consider it, but since it is a homebrew mapper that few emulators support, and no games actually use it, it would be very low on the priority list considering there are still official mappers yet to be emulated, and many bootleg mappers which would deserve more priority here.

**Comment by red031000** [comment]
of course, mappers with actual games that are used would be the priority, thanks for considering it though
